db/migrate/20170209203904_orchestration_stack_enhancements_for_ansible_service_provision.rb

```diff
@@ -0,0 +1,9 @@
+class OrchestrationStackEnhancementsForAnsibleServiceProvision < ActiveRecord::Migration[5.0]
+  def change
+    add_column :orchestration_stacks, :start_time,                   :timestamp
+    add_column :orchestration_stacks, :finish_time,                  :timestamp
+    add_column :orchestration_stacks, :configuration_script_base_id, :bigint
+    add_column :orchestration_stacks, :verbosity,                    :integer
+    add_column :orchestration_stacks, :hosts,                        :text, :array => true
```

Comment:
This is just for informational purposes, it can be displayed in the UI.  The hosts in AnsibleTower will be removed once the Job completes.